docs/sources/setup-grafana/configure-security/secretscan.md

```diff
@@ -0,0 +1,67 @@
+---
+aliases:
+  - /docs/grafana/latest/auth/secretscan/
+  - /docs/grafana/latest/setup-grafana/configure-security/secretscan/
+description: Detect and revoke leaked Grafana service account tokens
+title: Grafana Secret Scanning
+weight: 1000
+---
+
+# Grafana Secret Scanning
+
+Grafana allows you to check if your [Service account tokens]({{< relref "../../administration/service-accounts/" >}})
+are leaked in GitHub through the [GitHub Secret Scanning](https://docs.github.com/en/code-security/secret-scanning/about-secret-scanning) service.
+
+<div class="clearfix"></div>
+
+## Token Revocation
```

Comment:
I don't think a H2 is necessary here. The content in this section just continues what you've written above. Let's remove it.